internal/config/config.go

```diff
@@ -89,6 +102,16 @@ func (e Egress) validate() error {
 	return nil
 }
 
+func (r Route) validate() error {
+	if r.OrganizationID == "" {
+		return fmt.Errorf("organization ID is missing")
+	}
+	if len(r.Namespaces) == 0 && !r.ClusterResources {
+		return fmt.Errorf("no namespace or ClusterResource routing defined for the organization %s", r.OrganizationID)
+	}
```

Comment:
I'm not sure if we should error on this.
Yeah, 
```yaml
clusterScopedResources: false
namespaces: []
```
is kinda stupid, but it's not an error? 🤔 

(I'm not sure about this, erroring out might not be bad to make the customer aware? Really not sure 🤔 )

Reply:
Yeah, I was also thinking to let users know this will route nothing and it is an error. A bit weird behavior if you are able to create a no-op route.